src/stac_api_validator/validations.py

```diff
@@ -582,7 +596,10 @@ def validate_api(
 
     if not errors:
         try:
-            catalog = Client.open(root_url)
+            headers = {}
+            if r_session.headers and r_session.headers.get("Authorization"):
+                headers["Authorization"] = str(r_session.headers["Authorization"])
+            catalog = Client.open(root_url, headers=headers)
```

Comment:
Same question as in `get_catalog`, is there any reason to limit this copy to the `Authorization` header? Why not copy all headers?